public/pages/techtalk.html

```diff
@@ -51,53 +51,67 @@
-         With a B.A. and M.A. in American History, she decided to attend The Grace Hopper Program at Fullstack Academy to lean full stack JavaScript in 2017. Afterwards, she was a participant at Recurse Center, a programming retreat located in New York City.
-        </p>
+          <h2>Tech Talk: Building a Data Analytics Platform Using Google Cloud</h2>
+          <p>
+            Our Wednesday-night lab is online for now due to Covid! Join us February 16th from 6:30-7:30pm via Zoom using the link <a href="#zoom">below</a>!
+          </p>
+          <p>
+
+
+          </p>
+          <p>
+            Jean-Paul <a href="https://linkedin.com/in/mugizico">JP Mugizi</a> is a Senior Software Engineer at Chipper Cash, a tech company  focusing on building  financial products for the African continent.
+            He specializes in data platform and distributed systems engineering.
+            Previously, he was an engineer at Mailchimp, Capital One, Login.gov / 18F, Boeing and others.
+            He was born and raised in Kigali City, Rwanda but currently resides in Brooklyn, NY.
+            Favorite hobbies include music, reading, cycling, and traveling.
+          </p>
+          <div style="left; margin: 0 10px; padding: 10px;left;text-align: center;">
+          <img src="../img/TechTalk-JP.jpg" style="max-width: 250px"/>
+          <h4 style="margin: 0; text-align: center;">JP Mugizi</h4>
+          <p style="margin-top: 5px;">Senior Software Engineer @ Chipper Cash<br>Specializes in Data Platform &
+            Distributed Systems Engineering.</p>
+          </div>
+
+
       </div>
 
       <!-- navigation-component 1.1 -->
       <aside class="aside-navigation small patch">
         <h4>Upcoming Talks</h4>
         <ul>
-          <li><a class="current">Feb. 9 - Erin Shaw: Are Code Bootcamps a fast-track to a career in tech?</a></li>
-          <li><a >Feb. 16 - JP Mugizi: Building a Data Analytics Platform Using Google Cloud</a></li>
+            <li><a >Feb. 16 - JP Mugizi: Building a Data Analytics Platform Using Google Cloud</a></li>
+            <li><a >Feb. 23 - Wesley Hall </a></li>
         </ul>
         <h4>Past Talks</h4>
         <ul>
-          <li><a href='/pages/techtalk_no-code.html'>Feb 2. - Bubble.io and No-code Development / Paul Beresuita </a></li>
+          <li><a href='/pages/techtalk_Bootcamp.html' class="previous">Feb. 9 - Erin Shaw: Are Code Bootcamps a fast-track to a career in tech?</a></li>
+          <li><a href='/pages/techtalk_no-code.html' class="previous">Feb 2. - Bubble.io and No-code Development / Paul Beresuita </a></li>
           <li><a href='/pages/techtalk_artificial-intelligence.html'>Jan. 19 - Artificial Intelligence / Tyler Suard (1/19)</a></li>
         </ul>
       </aside>
     </div>
 
     <h2>Featured Tech Talk Speaker</h2>
     <div class="imgcontain">
-      <img src="../img/tech-talk_Erin-Shaw.png">
+      <img src="../img/tech-talk_jp mugizi.png">
```

Comment:
Don't change this now, but for the future it's best not to put spaces in filenames, some systems can have problems with that. I think we're fine because we're using GitHub and Netlify which can accept spaces. It's just a best-practice for when your files might land on a 3rd party server somewhere that's odd.

Reply:
Sorry, that one slipped out of my mind but his personal photo's name was right.